pyscf/gw/gw_cd.py

```diff
@@ -70,7 +71,7 @@ def kernel(gw, mo_energy, mo_coeff, Lpq=None, orbs=None,
     nvir = nmo-nocc
 
     # v_hf from DFT/HF density
-    if vhf_df and gw.frozen == 0:
+    if vhf_df: #and gw.frozen == 0:
```

Comment:
Is there a reason to keep "and gw.frozen == 0" in there if it's commented in? I've noticed in other places in pyscf and was wondering when commenting in is done.